@xen-orchestra/proxy/src/app/_debounceResource.js

```diff
@@ -1,6 +1,6 @@
 import Disposable from 'promise-toolbox/Disposable'
 
-export async function debounceResource(pDisposable, hooks, delay = 0) {
+export async function debounceResource(pDisposable, addDisposeListener = dispose => Function.prototype, delay = 0) {
```

Comment:
We should document `addDisposeListener` because it's not obvious why it's necessary (at least for me).